src/load.cpp

```diff
@@ -97,7 +98,7 @@ static void short_circuit_comp_init()
     COMP2->CSR |= COMP_INPUT_PLUS_IO2;
 
     // select VREFINT divider as negative input
-    COMP2->CSR |= COMP_INPUT_MINUS_VREFINT;
+    COMP2->CSR |= COMP_INPUT_MINUS_1_4VREFINT;
```

Comment:
During testing I realized that the short circuit protection with current low-pass filter triggered very late, partly because of this hardware issue, I guess https://github.com/LibreSolar/pwm-2420-lus/issues/1

So I reduced it, but it didn't have much effect. Even with lower comparator reference it didn't always trigger the short cirucit detection, that's why I also implemented the voltage dip detection. This setting will have to be further tuned in the future when the hardware issue is resolved.